### PR TITLE
Newer API Method should be encouraged

### PR DESCRIPTION
**Pull request by benravetta**
Changed authentication structure to show newer API method, over old username (which is still reference in the auth documentation for anyone using it) but the newer method should be encouraged.

### CODE REVIEW

File: api/sample-code.md [hunk 1]
```diff
@@ -28,14 +28,13 @@ $whmcsUrl = "https://www.yourdomain.com/path/to/whmcs/";
 // For WHMCS 7.2 and later, we recommend using an API Authentication Credential pair.
 // Learn more at http://docs.whmcs.com/API_Authentication_Credentials
 // Prior to WHMCS 7.2, an admin username and md5 hash of the admin password may be used.
-$username = "your_api_credential_identifier";
-$password = "your_api_credential_secret";
+$api_identifier = 'D4j1dKYE3g40VROOPCGyJ9zRwP0ADJIv';
```
Comment (mattpugh) [inline api/sample-code.md]: Thanks for the suggestion here.  Certainly happy to incorporate this. However could you replace the values in `$api_identifier` and `$api_secret` with placeholder values to make it clearer to other users these need to be populated with their own values?

File: api/sample-code.md [hunk 2]
```diff
@@ -28,14 +28,13 @@ $whmcsUrl = "https://www.yourdomain.com/path/to/whmcs/";
...
+$api_secret = 'F1CKGXRIpylMfsrig3mwwdSdYUdLiFlo';
 
-// Set post values
 $postfields = array(
-    'username' => $username,
-    'password' => $password,
-    'action' => 'GetClients',
+    'identifier' => $api_identifier,
+    'secret' => $api_secret,
+    'action' => $api_call,
```
Comment (mattpugh) [inline api/sample-code.md]: The `$api_call` variable doesn't appear to be defined anywhere in the sample code.

File: api/sample-code.md [hunk 3]
```diff
@@ -59,4 +58,7 @@ $jsonData = json_decode($response, true);
 
 // Dump array structure for inspection
 var_dump($jsonData);
+
+?>
```
Comment (mattpugh) [inline api/sample-code.md]: The closing PHP tag is not required in PHP files so could you remove these 3 new lines at the end of the code sample for us? Thanks!

**Comment by WHMCSPeter** [comment]
PR superseded by #234 to rectify comments.

**Pull request closed without merging**